.github/ISSUE_TEMPLATE/application.yml

```diff
@@ -168,7 +168,7 @@ body:
 - type: textarea
   attributes:
     label: Business Product or Service to Project separation
-    description: If this project is identical (name, features, etc.) or closely related to one or more products or services of the sponsoring company/organization(s), how do you plan to separate this project from any products in terms of organization and development? If it is not related to a product or service, just provide "N/A".
+    description: If this project is related to one or more products or services of the sponsoring company/organization(s), please explain how you plan to separate this project from any products in terms or organization and development.  The project could be the "upstream" open source version of a product or service, be a required dependency, or be a key part of the infrastructure you use to develop your products/services.  If the project is completely unrelated to any product/service, please put "This project is unrelated to any product."
```

Comment:
`terms or` should be `terms of`

Reply:
`to any product.` should be `to any product or service.`

Reply:
Updated.  Also added "or if this project is not being contributed by a business" which is a possible case, sometimes individuals submit projects.